zebra-state/src/service/finalized_state/disk_format/tests/snapshots/orchard_anchors_raw_data@mainnet_2.snap

```diff
@@ -0,0 +1,12 @@
+---
+source: zebra-state/src/service/finalized_state/disk_format/tests/snapshot.rs
+assertion_line: 125
+expression: cf_data
+
+---
+[
+  KV(
+    k: "ae2935f1dfd8a24aed7c70df7de3a668eb7a49b1319880dde2bbd9031ae5d82f",
```

Comment:
Question: all `orchard_anchors_raw_data` (and for Sprout and Sapling) seem to be the same. Just wanted to check if that's expected.

Reply:
The anchors are the same for all blocks on both networks until the first shielded transaction in each pool.
(This depends on the transparent maturity limit (sprout), or the pool being enabled by a network upgrade activation (sapling, orchard).)